client/lib/importer/importer-config.tsx

```diff
@@ -80,7 +80,8 @@ function getConfig( {
 				},
 			}
 		),
-		overrideDestination: '/migrate/%SITE_SLUG%',
+		overrideDestination:
+			'/setup/site-migration?siteSlug=%SITE_SLUG%&siteId=%SITE_ID%&ref=wp-admin-importer',
```

Comment:
I think we should use a more accurate `ref` value, as this is not technically coming from `wp-admin`. I would vote for something like `calypso-migrate`.
```suggestion
			'/setup/site-migration?siteSlug=%SITE_SLUG%&siteId=%SITE_ID%&ref=calypso-migrate',
```

Reply:
Ref updated here [63e9e1d](https://github.com/Automattic/wp-calypso/pull/90994/commits/63e9e1d829a38153c1059dbd12f341dd5fbc649a)